### PR TITLE
remove sortbykey

### PR DESCRIPTION
**Pull request by marsishandsome**
close https://github.com/tikv/migration/issues/21

`sortByKey` is too slow and it is not needed.
We can just delete it.

### CODE REVIEW

File: online-bulk-load/src/main/scala/org/tikv/bulkload/RawKVBulkLoader.scala [hunk 1]
```diff
@@ -116,40 +116,44 @@ class RawKVBulkLoader(tiConf: TiConfiguration, sparkConf: SparkConf) extends Ser
...
-        val pairsIterator = tiro2.map { keyValue =>
-          new Pair[ByteString, ByteString](ByteString.copyFrom(keyValue._1), ByteString.copyFrom(keyValue._2))
-        }.toList.sortWith((a, b) => Key.toRawKey(a.first).compareTo(Key.toRawKey(b.first)) < 0).toIterator.asJava
-        val tiSession = TiSession.create(tiConf)
+        var tiSession: TiSession = null
+        while(tiSession == null) {
+          try {
+            tiSession = TiSession.create(tiConf)
+          } catch {
+            case e: Throwable => logger.warn("create tiSession failed!", e)
```
Comment (liangyouze) [inline online-bulk-load/src/main/scala/org/tikv/bulkload/RawKVBulkLoader.scala:151]: If create TiSession something error, the process will never exit

Comment (marsishandsome) [reply online-bulk-load/src/main/scala/org/tikv/bulkload/RawKVBulkLoader.scala:151]: sure, i will fix it

Comment (marsishandsome) [reply online-bulk-load/src/main/scala/org/tikv/bulkload/RawKVBulkLoader.scala]: fixed. PTAL

**Comment by liangyouze** [comment]
looks good

**Pull request merged**